.zenodo.json

```diff
@@ -1,22 +1,29 @@
 {
   "license": "Apache-2.0",
-  "description": "<ul><li>Refactor SSL checks (#175). (Enol Fernández)</li><li>Refactor SSL checks (#175). (Enol Fernández)</li><li>Code Clean-up (#177). (Enol Fernández)</li><li>Import stdout (default one) and ams publishers (#178). (Enol Fernández)</li></ul>",
+  "description": "<ul> <li>Migrate from travis to GitHub Actions (#195, #198) (Enol Fernández)</li> <li>Reformatted code with black (#196) (Enol Fernández)</li> <li>Move templates inside the module (#194) (Enol Fernández)</li> <li>Publish project name and project domain name (#190) (Enol Fernández)</li> <li>Improve py3 compatibility (#189, #192) (Enol Fernández)</li> <li>Add EOSC-hub funding acknowledgement as requested by project (#188) (Enol Fernández)</li> <li>Update list of requirements for CentOS7 RPM building (#183) (Pablo Orviz)</li> </ul> ",
   "language": "eng",
-  "title": "EGI-Foundation/cloud-info-provider: 0.12.0",
-  "keywords": ["bdii", "grid", "information", "cloud", "EGI", "ams"],
+  "title": "EGI-Foundation/cloud-info-provider: 0.12.1",
+  "keywords": [
+    "bdii",
+    "grid",
+    "information",
+    "cloud",
+    "EGI",
+    "ams"
+  ],
   "grants": [
     {
       "id": "10.13039/501100000780::777536"
     }
   ],
   "upload_type": "software",
-  "version": "0.12.0",
+  "version": "0.12.1",
   "communities": [
     {
       "identifier": "egi-foundation"
     }
   ],
-  "publication_date": "2019-11-06",
+  "publication_date": "2020-11-18",
```

Comment:
This gets outdated soon if the release does not happen :(

Reply:
Let's close and redo?